package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "my-special-way-server",
-  "version": "0.0.14",
+  "version": "0.0.17",
   "description": "server-for-my-special-way-android/portal",
   "author": "my-special-way",
   "license": "MIT",
   "scripts": {
-    "build": "./scripts/build",
+    "build": "bash ./scripts/build",
```

Comment:
Travis runs scripts in sh instead of bash this is why I added a "." yesterday, it's the equivalent to source command in bash.
Otherwise env variables won't be exported. 
Is bash needed here? 
And need to check if it's still recognize the variables from prepare-push script